src/components/Button/index.js

```diff
@@ -108,6 +108,17 @@ const whiteStyle = css({
     color: '#000'
   }
 })
+const blackPrimaryStyle = css({
+  backgroundColor: '#000',
+  borderColor: '#000',
+  color: '#fff',
+  '@media (hover)': {
+    ':hover': {
+      backgroundColor: colors.secondary,
```

Comment:
if still needed it'd prefer a grey as hover bg instead of secondary:
`d3Color.lab('#000').brighter(3).toString()` -> `#818181`